src/layer/const.ts

```diff
@@ -60,6 +60,12 @@ export type PaginatedTiles = {
   hasMore: boolean;
 };
 
+export type Flyover = {
+  tiles: Tile[];
+  startTime: string;
+  endTime: string;
+};
```

Comment:
Thanks, changed as suggested (`fromTime` and `toTime` too).

I renamed it to `findFlyoverIntervals` because I find it more correct (it should return the interval, not a point in time for every flyover).

Reply:
Nice, even better! Looks good, resolving.